### PR TITLE
Fix duplicate entry errors (Fixes #40)

### LINKED ISSUE #40: Duplicate entry 'wDgK7x9W0yJ5joARRqmVlvQoRB48kb' for key 'value'
private $_alphabet = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890';  

Table = COLLATE=utf8_unicode_ci . 

hash entryid 32443 string(30) "wDgK7x9W0yJ5joA**R**RqmVlvQoRB48kb" 
hash entryid 75645 string(30) "wDgK7x9W0yJ5joA**r**RqmVlvQoRB48kb"

### PR DESCRIPTION
**Pull request by jurajkapsz**
If someone else could also test this, I would appreciate. Tested on MySQL v5.6 and MariaDB v10.1 .

### CODE REVIEW

File: extension.driver.php [hunk 1]
```diff
@@ -114,6 +114,20 @@ public function update($previousVersion = false)
              );
         }
 
+        if (version_compare($previousVersion, '2.0.2', '<')) {
```
Comment (jurajkapsz) [inline extension.driver.php]: Extension update version needs to be v2.0.2+ for this condition.

Comment (nitriques) [reply extension.driver.php]: Can you add the missing lines in the meta.xml file to create the new version please ?

Comment (jurajkapsz) [reply extension.driver.php]: @nitriques sure, like this 470fa5b ?

Comment (nitriques) [reply extension.driver.php]: Yup!

**Pull request merged**